src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/CashOutBaseRepository.cs

```diff
@@ -51,5 +52,21 @@ public string VolumeText
         }
 
         public string PreviousId { get; set; }
+
+        public double FeeSize { get; set; }
+        public FeeType FeeType { get; set; }
```

Comment:
Is this correct that both two properties are writable? I think one should be readonly as it derives from the second.

Reply:
Why? FeeType can be Absolute or Relative. This value can't be inferred from the value of FeeSize or vice versa.

Reply:
My comment was about FeeTypeText. Sorry for the confusion.

Reply:
You are right, removed setters.